### PR TITLE
doc(examples): use h3s for titles

### PR DESCRIPTION
**Pull request by redallen**
**What**: Towards https://github.com/patternfly/patternfly-org/issues/1794:
- Ensure all example md files are in `/examples` dir
- Replaced code block titles with `h3` titles. Caption support coming first to org, and then to React!

Towards https://github.com/patternfly/patternfly-org/issues/1793:
- Change frontmatter `title` to `id` which will be synced across Core, React, and Org

**Additional issues**:

### CODE REVIEW

File: packages/react-catalog-view-extension/src/components/PropertiesSidePanel/examples/PropertiesSidePanel.md [hunk 1]
```diff
@@ -17,7 +16,8 @@ This package is currently an extension. Extension components do not undergo the
 
 ## Examples
 
-```js title=Properties-Side-Panel-with-PropertyItems
+### Properties side panel
```
Comment (jschuler) [inline packages/react-catalog-view-extension/src/components/PropertiesSidePanel/examples/PropertiesSidePanel.md:19]: some text fell off

File: packages/react-charts/src/components/ChartArea/examples/ChartArea.md [hunk 2]
```diff
@@ -173,7 +174,8 @@ class BottomAlignedLegend extends React.Component {
 }
 ```
 
-```js title=Multi--color-(unordered)-bottom--left-aligned-legend-and-responsive-container
+### Multi-color (unordered) bottom  left aligned legend and responsive container
```
Comment (jschuler) [inline packages/react-charts/src/components/ChartArea/examples/ChartArea.md:177]: guessing the previous double dash was an escaped single dash? then this should say `bottom-left`

File: packages/react-charts/src/components/ChartBar/examples/ChartBar.md [hunk 3]
```diff
@@ -98,7 +99,8 @@ PurpleBottomLegend = (
 )
 ```
 
-```js title=Multi--color-(ordered)-with-bottom--left-aligned-legend
+### Multi-color (ordered) with bottom  left aligned legend
```
Comment (jschuler) [inline packages/react-charts/src/components/ChartBar/examples/ChartBar.md:102]: `bottom-left`

**Comment by redallen** [comment]
@jschuler just checked and it looks like I got them all.

**Pull request merged**